charts/atlas-operator/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: mongodb-atlas-operator
+description: |-
+  MongoDB Atlas Operator - a Helm chart for installing and upgrading Atlas Operator: the official Kubernetes operator allowing to manage MongoDB Atlas resources from Kubernetes
+type: application
+version: 0.4.0
```

Comment:
this is Helm chart version. Could we make it 0.1.0? Synchronizing with App would make it difficult to make fixes outside of the release schedule.

Reply:
yeah, makes sense 👍 

I think the same should be done to the CRDs as well